### PR TITLE
Changes for final room version

### PR DESCRIPTION
**Pull request by stevenzicka**

### CODE REVIEW

**Review by Jedimasterjohns — CHANGES_REQUESTED** [review]
I need to run through this again but unfortunately when I got to the end of the escape room I was not able to exit. I wonder if something in your code broke the handleEscape function that is passed up from Room -> EscapeRoom -> RoomCanvas. I also did not see a back button.

File: src/components/EscapeRoom/Room.jsx [hunk 1]
```diff
@@ -560,7 +538,7 @@ export default function Room({ handleEscape, setIsRoomLoaded })
                 setExitDoor(true)
                 break;
             case intersectsArray.includes('Exit_Screen') && exitDoor:
-                handleEscape();
+                console.log("You Escaped!");
```
Comment (Jedimasterjohns) [inline src/components/EscapeRoom/Room.jsx:541]: This needs to be changed back to handleEscape()

Comment (heather-fillerup-miller) [reply src/components/EscapeRoom/Room.jsx:541]: Also need to change line 12 back to export default function Room({ handleEscape, setIsRoomLoaded })

image

**Review by Jedimasterjohns — APPROVED** [review]

**Review by heather-fillerup-miller — APPROVED** [review]
Approved

**Pull request merged**